[1.x] Please install .pc file for easy discoverability

The ustream project that uses janus fails to find the janus headers:
In file included from src/acap.c:43:
In file included from src/logging.h:23:
/usr/local/include/janus/plugins/plugin.h:162:10: fatal error: 'refcount.h' file not found
  162 | #include "refcount.h"
      |          ^~~~~~~~~~~~
1 error generated.


This is because janus doesn't provide any way to discover it or to determine what CFLAGS does it need.

The acrual reason is that this CFLAGS argument is missing: -I$(PREFIX)/include/janson.

CFLAGS and LDFLAGS should be supplied by the .pc file so that ```pkg-config --cflags --libs janus``` would print all necessary flags.

--- a/src/plugins/plugin.h
+++ b/src/plugins/plugin.h
@@ -159,7 +159,7 @@ janus_plugin *create(void) {
 
 #include <glib.h>
 
-#include "refcount.h"
+#include "../refcount.h"
 
 
 /*! \brief Version of the API, to match the one plugins were compiled against

--- a/src/transports/transport.h
+++ b/src/transports/transport.h
@@ -93,7 +93,7 @@ janus_transport *create(void) {
 #include <glib.h>
 #include <jansson.h>
 
-#include "refcount.h"
+#include "../refcount.h"
 
 
 /*! \brief Version of the API, to match the one transport plugins were compiled against */

